_posts/2021-03-04-geoserver-2-19-rc-released.md

```diff
@@ -0,0 +1,240 @@
+---
+author: Andrea Amie, Jody Garnett
+date: 2021-03-04
+layout: post
+title: GeoServer 2.19-RC Released
+categories:
+- Announcements
+tags:
+- Release Candidate
+---
+
+We are happy to announce GeoServer [2.19-RC](/release/2.19-RC/) release candidate is available for testing. Downloads are available ([zip](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/geoserver-2.19-RC-bin.zip/download) and [war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/geoserver-2.19-RC-war.zip/download)) along with [docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/geoserver-2.19-RC-htmldoc.zip/download) and [extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/extensions/).
+
+This is a GeoServer release candidate made in conjunction with GeoTools 25-RC and GeoWebCache 1.19-RC.
+
+  * Release candidates are a community building exercise and are not intended for production use.
+  * We ask the community (everyone: individuals, organizations, service providers) to download and thoroughly test this release candidate and report back.
+  * Participating in testing release candidates is a key expectation of our [open source social contract](http://www.ianturton.com/talks/foss4g.html#/). We make an effort to thank each person who tests in our release announcement and project presentations!
+  * GeoServer [commercial service providers](http://geoserver.org/support/) are fully expected to test on behalf of their customers.
+
+## Release Candidate Testing Priorities
+
+This is an exciting release and a lot of great new functionality has been added. We would like to ask for your assistance testing the following:
+
+  * The number one testing priority is to try out GeoServer with your data! _Mass market open source thrives on having many people to review. Scientific open source like GeoServer thrives on exposure to many datasets_.
+  * Help check that new [extension download bundles](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/extensions/) have contain everything needed, including appropriate readme instructions and open source license information.
+  * The rest of this blog post highlights new features for GeoServer 2.19, please try out these features, read the documentation links, and ask questions.
+
+Known Issues:
+
+  * Layer configured with missing style throws NPE
+
+## MapML extension
+
+The MapML plugin has been under developer for some time as a community module and is now ready for a wider audience.
```

Comment:
Map Markup Language (MapML) is a proposed extension to HTML for maps.  MapML.  The format is developed in public with collaboration from OGC and W3C members and the public in the Maps for HTML Community Group. The objective is to extend browsers native functionality to support accessible, performant, semantic Web maps.  The GeoServer MapML extension will closely track the specification as it evolves.  Please join us! Find out more at https://maps4html.org/